### ISSUE TITLE
Waterfall direction opposite of most digi apps

### ISSUE THREAD

**Issue by Jmelz** [df8oe/mchf-github#1069]
Has there been any thought to reversing the waterfall direction? Most other applications use a waterfall that scrolls down (like an actual waterfall).

If the issue is that the consensus is to keep the frequency counters next to the spot where the waterfall 'starts' from, why not move those to the top as well then?

**Comment by df8oe** [comment]
...because that is much work and it is working "as it is" too, or not ;)

**Comment by Jmelz** [comment]
Thanks for the reply. It was just unusual to change in my head a waterfall from falling down to falling up :)

**Comment by Jmelz** [comment]
I saw this today:

https://www.youtube.com/watch?v=HzEP_i8pxCs

Looks like someone was able to reverse the direction and then completely changed the layout as well to suit the larger screen they used.

**Comment by df8oe** [comment]
Yes. But that is not mcHF - it is Raspberry Pi powered and completely other hardware.

**Comment by Jmelz** [comment]
Really? I read this in the description: " Ui board same as for original mcHF, mcu STM32F405, except one additional adapter board between LCD and UI board "

**Comment by df8oe** [comment]
Really! STM32F4 does not have enough RAM to hold waterfall AND spectrum data of larger LCD.

But: not a problem to reverse it. If you do have much time and the knowledge: you are invited to do it.

I will not waste my time for things which are not functional issues but only optical ones. It is a community project - just start over...

**Comment by db4ple** [comment]
Hi Andreas,
I don't think, you are right. The text says "3.5 RPi Display" not RPi and the youtube user says modified screen routines (much faster) but using the 405. So it is the genuine thing. Hopefully the youtube user will release the source code and will not just sit on it so that we can see how he did this trick. 
There is enough RAM if you leave out FreeDV I would say. Maybe even without this.

73
Danilo

**Comment by Jmelz** [comment]
Yeah I figured as much with displaying both. Not interested in that, just the waterfall direction. I'm a complete novice with code - I typically beta test for several projects, and got into the mchf for the low cost and speedy updates you provide. I am quite incapable of making any useful changes. 

No worries on the topic - I just wanted to say that it looks like it could be done. I figured many people would be interested in having this display the waterfall correctly, but if there is not interest, and no one is pushing (or even helping with the code), then I can see it is not worth the time right now.
